<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="2" hostname="AKASH" failures="1" tests="1" name="Default test(failed)" time="0.053" errors="0" timestamp="2021-03-31T11:05:49 IST">
  <testcase classname="failedTestNG.Sample1" name="m3" time="0.011">
    <failure type="java.lang.AssertionError" message="null">
      <![CDATA[java.lang.AssertionError: null
at failedTestNG.Sample1.m3(Sample1.java:24)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- m3 -->
  <testcase classname="failedTestNG.Sample1" name="m1" time="0.0">
    <ignored/>
  </testcase> <!-- m1 -->
  <testcase classname="failedTestNG.Sample1" name="m2" time="0.0">
    <ignored/>
  </testcase> <!-- m2 -->
</testsuite> <!-- Default test(failed) -->
